src/v2/models/AppState.js

```diff
@@ -54,10 +54,10 @@ export default Model.extend({
     showSignoutLink: {
       deps: ['idx'],
       fn: function (idx = {}) {
-        const invalidSignOutSteps = ['IDENTIFY', 'ENROLL', 'SUCCESS'];
-        // hide signout for IDENTIFY, ENROLL & SUCCESS step
+        const invalidSignOutSteps = ['IDENTIFY', 'LOGIN', 'ENROLL', 'SUCCESS'];
+        // Hide signout for IDENTIFY, LOGIN, ENROLL & SUCCESS intents
         return idx.actions
-          && _.isFunction(idx.actions.cancel) && !invalidSignOutSteps.includes(idx.context.step);
+          && _.isFunction(idx.actions.cancel) && !invalidSignOutSteps.includes(idx.context.intent);
```

Comment:
Seems like there's no `step` but `intent` instead. Please let me know if I'm missing something here.

Reply:
I don't think. `step` has been removed. `intent` isn't sufficient given `signout` is also more ux concern.
e.g. `identify` has cancel but I doubt we want to show `signout` link.
I'm thinking eventually we have to rely on remediation form name.
any thoughts @nikhilvenkatraman-okta

Reply:
yes ,we can no longer use `step` and I dont think `intent` is right either. We need to change this logic to use the form names instead. Should we handle that in a different ticket and keep this the same here for now ?

Reply:
It makes sense to open another ticket - [OKTA-305970](https://oktainc.atlassian.net/browse/OKTA-305970) - keeping this as is in this PR. We can use the new ticket for verifying all the screens as well.